src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.PragmaHelpers.cs

```diff
@@ -169,6 +169,75 @@ internal static SyntaxToken GetNewEndTokenWithAddedPragma(SyntaxToken endToken,
                     return endToken.WithTrailingTrivia(trivia.InsertRange(index, pragmaTrivia));
                 };
             }
+
+            internal static void NormalizeTriviaOnTokens(AbstractSuppressionCodeFixProvider fixer, ref Document document, ref SuppressionTargetInfo suppressionTargetInfo)
```

Comment:
Method just moved into this helper class from PragmaRemoveAction, as it is now also used by add pragma action.